environment.yml

```diff
@@ -2,23 +2,42 @@
 name: esmvaltool
 channels:
   - conda-forge
+  - nodefaults
 
 dependencies:
   - cartopy
   - cf-units>=3.0.0,<3.1.0  # github.com/ESMValGroup/ESMValCore/issues/1655
   - cftime
+  - dask[array]
```

Comment:
This syntax for specifying optional extra dependencies is not supported by conda as far as I'm aware? https://github.com/conda/conda/issues/7502

Reply:
this was a bit of a shot in the dark to me, I left it like this since it didn't fail, but cheers for pointing that issue out - no need to have it like this unless we think this will be implemented in conda in the future

Reply:
If all the tests pass fine, I think the conda packages already include all required extras, so the square bracket bits can just be left out.

Reply:
done